src/routes/product/[handle].svelte

```diff
@@ -184,12 +184,13 @@
             <div
               class="group relative block aspect-square overflow-hidden border border-white/20 bg-zinc-800/50"
             >
-              <a sveltekit:prefetch href={`/product/${product.node.handle}`}>
```

Comment:
Should this still be a link?

Reply:
not anymore, GridTile was refactored to be a link and it accepts the href as a prop, so that is what we're using now